source/platforms/github/_tests/_github_git.test.ts

```diff
@@ -20,15 +20,19 @@ export const requestWithFixturedContent = async (path: string): Promise<() => Pr
   Promise.resolve(readFileSync(`${fixtures}/${path}`, {}).toString())
 )
 
+const pullRequestInfoFilename = "github_pr.json"
+const masterSHA = JSON.parse(readFileSync(`${fixtures}/${pullRequestInfoFilename}`, {}).toString()).base.sha
```

Comment:
I thought maybe it had something to do with line endings, but that shouldn't be an issue: https://github.com/facebook/jest/pull/1879